src/main/webapp/scripts/map.js

```diff
@@ -176,7 +176,7 @@ function placeMarkerAndPanTo(latLng, map) {
   });
   /* Recenter the Map */
   map.panTo(latLng);
-  togglePopup(latLng);
+  togglePopup(latLng, marker);
```

Comment:
I guess togglePopup function is not updated yet? since it currently only takes latlng as a parameter. Also, may I know why do we need to pass the marker? is it to change the icon when the recommendation is submitted (or remove it when recommendation is not submitted)?

Reply:
This is updated in the next code version (see CL #29). Thanks for pointing it out though - great spot!